.gitmodules

```diff
@@ -18,3 +18,6 @@
 [submodule "src/liblibc"]
 	path = src/liblibc
 	url = https://github.com/rust-lang/libc.git
+[submodule "src/doc/nomicon"]
+	path = src/doc/nomicon
+	url = https://github.com/rust-lang-nursery/nomicon
```

Comment:
None of us felt super strongly, but @brson said, with regards to the reference, that it needs a lot of work. The same is true for the nomicon; it still has the huge header at the top, for example.

Given that, the nursery feels appropriate, though I can see putting it in `rust-lang` too!